backend/api/tests/asset_factory.py

```diff
@@ -181,6 +181,9 @@ def get_log_permissions(owner: str, public: bool) -> dict:
 def get_computetask_dates(status: int, creation_date: datetime.datetime) -> tuple[datetime, datetime]:
     start_date = end_date = None
     if status in (
+        ComputeTask.Status.STATUS_BUILDING,
+        ComputeTask.Status.STATUS_WAITING_FOR_PARENT_TASKS,
+        ComputeTask.Status.STATUS_WAITING_FOR_EXECUTOR_SLOT,
```

Comment:
Do we want to start the execution date if waiting ?

Reply:
Cause we didn't considered the status TODO previously, which seems to be the equivalent of `WAITING_FOR_EXECUTOR_SLOT`. And waiting for a parent task would not be considered as the execution of the task itself imo.

Reply:
The `TODO`from before includes `STATUS_BUILDING`, `STATUS_WAITING_FOR_PARENT_TASKS`, `STATUS_WAITING_FOR_EXECUTOR_SLOT`. The choice I made was to start the compute task when it starts building, I can understand this could be challenged, what do you think @SdgJlbl

Reply:
I can see an option where we only keep the execution part in this timer, and keep track of the building time separately, and that way, show both of these in the frontend task profiling. Or we can decide that what matters most is the "real" time spent (a bit like user time can be different from clock time when profiliing code). 
I might be wrong but I feel like it's something to keep in mind, but it does not matter too much right now, given that we are going to revamp the task profiling part very soon (unless the start time is also used for some other feature I missed).

Reply:
I keep it that way and we would act when we refactor building time if everyone agree